src/dmd/builtin.d

```diff
@@ -226,15 +350,15 @@ public extern (C++) void builtin_init()
     add_builtin("_D4core4math4sqrtFNaNbNiNfeZe", &eval_sqrt);
     add_builtin("_D4core4math4fabsFNaNbNiNfeZe", &eval_fabs);
     add_builtin("_D4core4math5expm1FNaNbNiNfeZe", &eval_unimp);
-    add_builtin("_D4core4math4exp21FNaNbNiNfeZe", &eval_unimp);
+    add_builtin("_D4core4math4exp2FNaNbNiNfeZe", &eval_unimp);
```

Comment:
`&eval_unimp` => `&eval_exp2`

Reply:
There isn't an `exp2` function in `core.math`. We could add to it, but that discussion is beyond the scope of this PR.

Reply:
So you noticed the wrong mangle & fixed it instead of just removing that useless entry?

Reply:
Yes. I don't know why the eval_unimp entries are there, I didn't write that code. But I felt it was beyond the scope of this PR to address that.

Reply:
FWIW, the same goes for the `@trusted`/`@safe` overloads, the functions are all `@safe` IIRC.